apps/examples/wifi_manager_sample/wm_test.h

```diff
@@ -35,4 +35,6 @@ struct wt_options {
 	int scan_specific;
 	uint32_t stress_tc_idx;
 	uint8_t power_mode;
+	char *security;
```

Comment:
wt_options includes auth_type and crypto_type. I think security field is duplicated.